GoogleDataTransport/GDTCORLibrary/GDTCORUploadCoordinator.m

```diff
@@ -140,28 +140,18 @@ - (void)uploadTargets:(NSArray<NSNumber *> *)targets conditions:(GDTCORUploadCon
  * @return The current upload conditions.
  */
 - (GDTCORUploadConditions)uploadConditions {
-#if TARGET_OS_WATCH
-  // Assume connected on watchOS.
-  return GDTCORUploadConditionUnclearConnection;
-#else
-  SCNetworkReachabilityFlags currentFlags = [GDTCORReachability currentFlags];
-  BOOL reachable =
-      (currentFlags & kSCNetworkReachabilityFlagsReachable) == kSCNetworkReachabilityFlagsReachable;
-  BOOL connectionRequired = (currentFlags & kSCNetworkReachabilityFlagsConnectionRequired) ==
-                            kSCNetworkReachabilityFlagsConnectionRequired;
-  BOOL networkConnected = reachable && !connectionRequired;
-
+  GDTCORNetworkReachabilityFlags currentFlags = [GDTCORReachability currentFlags];
+  BOOL networkConnected = GDTCORReachabilityFlagsReachable(currentFlags);
   if (!networkConnected) {
     return GDTCORUploadConditionNoNetwork;
   }
-
+  // Assume always wifi connection on watchOS.
```

Comment:
This comment seems out of place--there's no mention or apparent watchOS context here.

Reply:
Removed